model/message.go

```diff
@@ -27,9 +27,8 @@ type Message struct {
 	Message string `form:"message" query:"message" json:"message" binding:"required"`
 	// The title of the message.
 	//
-	// required: true
 	// example: Backup
-	Title string `form:"title" query:"title" json:"title" binding:"required"`
```

Comment:
The application name should be used as default value otherwise the title just would be empty.